### ISSUE TITLE
Add colorable to cssstats

### ISSUE THREAD

**Issue by mrmrs** [cssstats/cssstats#142]
We should possibly scrape for background / color combos in classes and report which ones fail accessibility. 

We should also add output for all possible combinations for colors present in a css file ala http://jxnblk.com/colorable

https://www.npmjs.com/package/colorable

**Comment by mrmrs** [comment]
Colorable feature set added in
